components/Promotion/index.rst

```diff
@@ -4,7 +4,9 @@ Promotion
 Super-flexible promotions system with support of complex rules and actions. Coupon codes included!
 
 .. toctree::
-   :numbered:
+   :maxdepth: 3
```

Comment:
You might have missed the argument about this, after some time and a lot of blood spilled, we 'agreed', to leave the depth at 2.